### DISCUSSION TITLE
Starting with v3.8.0, the macOS version will ship unsigned

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#413] [Announcements]
Unfortunately, I can no longer pay for the Apple Developer Program in order to sign applications, I simply don't have a way to pay.  Therefore, starting with version 3.8.0, the macOS version will ship unsigned. 

Launching the application: 

1. Go to System Preference
2. Click Security & Privacy
3. Select the Allow button under the Allow apps downloaded from section

**Comment by ararezaee** [comment]
No problem, just don't stop building it cause its an absolute gem.
And thanks for everything.

**Comment by faithk7** [comment]
![Screenshot 2024-11-23 at 9 29 13 PM](https://github.com/user-attachments/assets/32dfeaa2-0dc4-46b6-9e50-35a5b3fcabbc)

On macOS 15.2, you can open the app by first trying to launch it (it will be blocked) and then going to the settings Privacy & Security to click “Open Anyway.”

**Comment by keshav-ram-citytech** [comment]
Previously, it used to show the "Open Anyway" button for blocked applications. But, I'm not able to find it now.
I'm on macOS Tahoe Version 26.0.1 (25A362).

Edit: Nevermind! The Option 2 from [troubleshoot](https://github.com/massCodeIO/massCode?tab=readme-ov-file#troubleshooting) section helped me.

**Comment by dsoundmn** [comment]
> Unfortunately, I can no longer pay for the Apple Developer Program in order to sign applications, I simply don't have a way to pay. Therefore, starting with version 3.8.0, the macOS version will ship unsigned.

Might be worth considering https://www.macadmins.org/news/mac-admins-open-source

it looks like they have a solution for free.
